src/constants/memory/wram.asm

```diff
@@ -1724,6 +1724,13 @@ wD153::
 wD154::
   ds 1 ; D154
 
+;
+; This section is used for:
+; - Link's position history (during gameplay)
+; - Photo album display and print
+;
```

Comment:
The Photo album uses WRAM normally assigned to Link's position history during world gameplay.

(Makes sense, as the Photo Album and the world gameplay will never be displayed simultaneously).

Reply:
It's not doing something on the other WRAM bank? (I don't expect it will use the other WRAM banks often, as the stack is located on that address space)

Reply:
No, there's only a single RAM bank switch in this whole file, and it is very minor.

I guess that's because the photo album has to work on the DMG too (and the loading of photo attributes is probably done right from the actual data, rather than transiting through WRAM)